rb/lib/selenium/webdriver/remote/w3c_capabilities.rb

```diff
@@ -79,7 +79,7 @@ def firefox(opts = {})
             opts[:browser_version] = opts.delete(:version) if opts.key?(:version)
             opts[:platform_name] = opts.delete(:platform) if opts.key?(:platform)
 
-            new({browser_name: 'firefox'}.merge(opts))
+            new({browser_name: 'firefox', marionette: true}.merge(opts))
```

Comment:
Yeah I believe that's right. I originally updated both capabilities.rb & w3c_capabilities.rb files so that they had the correct option (since they are used by remote). But in reality only capabilities.rb needs `marionette: false`. However, we can't add that yet since remote geckodriver currently uses capabilities.rb. For now, users will have to use `Selenium::WebDriver::Remote::Capabilities.firefox(marionette: false)` even though they **shouldn't** need to pass that once we get w3c & remote married